compiler/rustc_middle/src/macros.rs

```diff
@@ -62,9 +62,9 @@ macro_rules! CloneTypeFoldableImpls {
                 fn super_visit_with<F: $crate::ty::fold::TypeVisitor<$tcx>>(
                     &self,
                     _: &mut F)
-                    -> bool
+                    -> ::std::ops::ControlFlow<(), ()>
                 {
-                    false
+                    ::std::ops::ControlFlow::CONTINUE
```

Comment:
is there a reason for using `::std::ops::ControlFlow` here and `ControlFlow` further down?